### PR TITLE
[DO-2462]Block the Platform for displaying on mobile gadgets

### PR DESCRIPTION
**Pull request by AlenaYanish**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -29,6 +30,26 @@ const App = () => {
       });
   }, []);
 
+  const [showMobileWarning, setShowMobileWarning] = useState(false);
```
Comment (roman-tiukh) [inline src/App.jsx:33]: move hook to top of component

File: src/App.jsx [hunk 2]
```diff
@@ -29,6 +30,26 @@ const App = () => {
...
+
+  const checkWindowWidth = () => {
+    if (window.innerWidth < 780) {
+      setShowMobileWarning(true);
+    } else setShowMobileWarning(false);
+  };
```
Comment (roman-tiukh) [inline src/App.jsx:39]: create this function in useEffect

Comment (roman-tiukh) [inline src/App.jsx:39]: this function must be wrapped into `throttle` or `debounce`

File: src/App.jsx [hunk 3]
```diff
@@ -29,6 +30,26 @@ const App = () => {
       });
   }, []);
 
+  const [showMobileWarning, setShowMobileWarning] = useState(false);
+
+  const checkWindowWidth = () => {
+    if (window.innerWidth < 780) {
+      setShowMobileWarning(true);
+    } else setShowMobileWarning(false);
```
Comment (roman-tiukh) [inline src/App.jsx:38]:
```suggestion
    } else {
      setShowMobileWarning(false);
    };
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/locales/uk/translation.json [hunk 4]
```diff
@@ -417,6 +417,7 @@
   "viewTheRegistry": "Перегляд реєстру",
   "views": "Переглядів",
   "warning": "Попередження",
+  "websiteAvailableOnDesktopDevices": "Перепрошуємо, платформа не доступна на даному екрані.",
```
Comment (roman-tiukh) [inline src/locales/uk/translation.json:420]: Not clear text, how should the user understand that the platform is not available on mobile devices only?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/locales/uk/translation.json (hunk 4 above)
Comment (AlenaYanish) [reply src/locales/uk/translation.json]: Change to "Sorry, this website isn`t available on mobile devices. Please open a desktop version."

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**